docs/manual-CN/2.query-language/4.statement-syntax/2.data-query-and-manipulation-statements/return-syntax.md

```diff
@@ -0,0 +1,35 @@
+# Return 语法
+
+Return 语句用于返回条件成立时的结果。
+
+```ngql
+    RETURN <var_ref> IF <var_ref> IS NOT NULL
+```
+
+* <var_ref> 变量名称， 示例： $var
+
+# 示例
+
+```
+(user@127.0.0.1) [myspace_test2]> $A = GO FROM 200 OVER like YIELD like._dst AS dst; \
```

Comment:
nebula>...

Reply:
I agree

(user@127.0.0.1) [myspace_test2]  is too long & meanless to read. can you just modify  to 

nebula>